### PR TITLE
[ISSUE#3249]Resolve the issues of codestyle of nacos-config module for phase8.

### PR DESCRIPTION
**Pull request by zongtanghu**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

(1).com.alibaba.nacos.config.server.service.capacity.
(2).com.alibaba.nacos.config.server.service.datasource.

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/service/capacity/CapacityService.java [hunk 1]
```diff
@@ -181,7 +149,7 @@ private void correctTenantUsage() {
             try {
                 Thread.sleep(100);
             } catch (InterruptedException e) {
-                // ignore
+                // Ignore.
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/capacity/CapacityService.java:152]: do not use comment to ignore, change the exception name as ignore, or ignored just be ok.

File: config/src/main/java/com/alibaba/nacos/config/server/service/capacity/CapacityService.java [hunk 2]
```diff
@@ -190,12 +158,49 @@ private void correctTenantUsage() {
...
+                    autoExpansion(targetId, null);
+                }
+            }
+            if (list.size() < INIT_PAGE_SIZE) {
+                break;
+            }
+            try {
+                Thread.sleep(100);
+            } catch (InterruptedException e) {
+                // ignore
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/capacity/CapacityService.java:190]: same question as above.

**Comment by zongtanghu** [comment]
I have already fixed the issue that you mentioned above. @KomachiSion

**Comment by KomachiSion** [comment]
Refer #3249

**Pull request merged**